server/controllers/filestore.controller.js

```diff
@@ -53,11 +53,6 @@ class FileStoreController {
         });
     }
 
```

Comment:
add this fn here also in commented form and mention that which fn to be used instead of this